docs/source/installation_instructions.rst

```diff
@@ -74,18 +50,31 @@ start menu and running the following line.
 If the Miniconda installation was successful, a list of available packages will appear.
 
 
-Pytentiostat Package Installation
+pytentiostat Package Installation
 ----------------------------------
 
-Once python and the conda are installed, the pytentiostat package installation is simple. Follow these steps to complete
-the Pytentiostat installation!
+Once the miniconda installation is complete, proceed with the pytentiostat package installation as follows.
 
 1. Open the anaconda prompt from the start menu.
-2. Navigate to the directory you would like to install the software, e.g. desktop
-3. In the command line, type:
+2. Create a conda environment where *mycondaenv* is the name you assign to your environment and *x.x* is the version of Python
+   you would like to use (the available list of python versions can be seen by typing conda search "^python$"). To create
+   the conda environment, type:
+
+.. code-block::
+
+   conda create -n mycondaenv python=x.x anaconda
+
+3. Type *y* to install the python version and the anaconda packaged libraries in the path_to_your_anaconda_location/anaconda/envs/mycondaenv.
+4. Activate the environment you created.
+
+.. code-block::
+
+   conda activate mycondaenv
```

Comment:
on windows I think this may be ``source activate mycondaenv`` so it is probably worth mentioning that.

Also, in documentation where I want people to replace something with their own thing, I usually put it between less-than -- greater than signs, and explain what that means to the user, so it would look like ``conda activate <mycondaenv>``.  Not sure if we want to use that convention here, but I like it.

Reply:
conda activate worked for me on windows through the anaconda prompt, however, I have not started trying to use the windows command prompt yet. I'm not very well-versed on using the different prompts.

I like that convention though, and I am implementing it in the next commit.